dev/io.openliberty.concurrent.internal.cdi/resources/io/openliberty/concurrent/internal/cdi/resources/CWWKCMessages.nlsprops

```diff
@@ -24,5 +24,33 @@
 #   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
 #   (single quote must be coded as one single quote '). 
 
-# All messages must use the range CWWCK1410 to CWWCK1499
+# All messages must use the range CWWKC1410 to CWWKC1499
 
+CWWKC1410.schedule.lacks.seconds=CWWKC1410E: The Schedule, {0}, specifies an \
+ empty value for the seconds at which to run. You must specify a non-empty value. \
+ Alternatively, omit the seconds from the Schedule and rely on the default \
+ behavior of scheduling the task to run at the second at the start of the minute.
+CWWKC1410.schedule.lacks.seconds.explanation=The empty array value indicates that \
+ the field must be disregarded when computing scheduled times. The seconds field \
+ cannot accept an empty value because it is the most granular unit and cannot be \
+ disregarded.
+CWWKC1410.schedule.lacks.seconds.useraction=Modify the seconds field of the \
+ Schedule annotation to specify a non-empty value, or omit the seconds field \
+ from the Schedule.
+
+CWWKC1411.qualified.res.err=CWWKC1411E: An error occurred creating a {0} bean \
+ for the {1} annotation or {2} deployment descriptor element with the \
+ {3} name and {4} qualifiers that is defined in the {5} application artifact. \
+ The error is: {5}
+CWWKC1411.qualified.res.err.explanation=An error occurred when creating the \
+ resource or attempting to register it as a bean.
+CWWKC1411.qualified.res.err.useraction=Ensure that the resource is configured \
+ correctly.
+
+CWWKC1412.qualifier.conflict=CWWKC1412E: The {0} application artifact includes \
```

Comment:
The {0} application artifact includes multiple {1} resource definitions that have the {2} qualifiers. The names of the conflicting {1} resource definitions are {3}.

Each resource definition of the same type that has a qualifier must have a different qualifier.

Ensure that each resource definition of the same type has a different qualifier or does not have any qualifiers.

Reply:
Updating as suggested